centos-ci/bootstrap.sh

```diff
@@ -51,5 +51,5 @@ export DISPLAY=:19
 
 
 echo "$(print_date): running integration tests"
-time scl enable rh-python35 sclo-vagrant1 -- sh -xc "cd ../.. && pipenv shell 'cd integration-tests && behave --no-color --junit; exit \\\$?'"
+time scl enable rh-python35 sclo-vagrant1 -- sh -xc 'cd ../../integration-tests && behave --no-color --junit; exit \\\$?'
```

Comment:
Yup, seeing your last commit there must be something funny going on since:

```bash
$ sh -xc 'true ; exit $?'
+ true
+ echo 0
0
$ sh -xc 'false ; echo $?'                                                                                         
+ false
+ echo 1
1
```

I suspect that SCL is doing some black magic under the hood:
```bash
$ scl enable rh-python35 -- sh -xc 'false ; exit $?'
+ false
+ exit 0
$ scl enable rh-python35 -- sh -xc 'false ; exit \$?'
+ false
+ exit 1
```
Weird stuff :)